src/client/components/EmployersQuestions/index.js

```diff
@@ -62,6 +62,7 @@ function EmployerQuestions(props) {
     return (
       <Form.Group controlId={props.employerData.id + name}>
         <Form.Label>{t(tk(name))}</Form.Label>
+        <Form.Text muted>{t(tk(`help-${name}`))}</Form.Text>
```

Comment:
Sorry for missing this!

Reply:
No problem! You didn't miss it, it was just added!